integrations/java-iceberg-1.5/openhouse-java-itest/src/test/java/com/linkedin/openhouse/javaclient/OpenHouseTableOperationsTest.java

```diff
@@ -0,0 +1,243 @@
+package com.linkedin.openhouse.javaclient;
+
+import static org.mockito.Mockito.*;
+
+import com.linkedin.openhouse.gen.tables.client.api.SnapshotApi;
+import com.linkedin.openhouse.gen.tables.client.api.TableApi;
+import com.linkedin.openhouse.gen.tables.client.model.CreateUpdateTableRequestBody;
+import com.linkedin.openhouse.gen.tables.client.model.Policies;
+import com.linkedin.openhouse.gen.tables.client.model.PolicyTag;
+import com.linkedin.openhouse.gen.tables.client.model.Retention;
+import com.linkedin.openhouse.relocated.org.springframework.web.reactive.function.client.WebClientRequestException;
+import com.linkedin.openhouse.relocated.org.springframework.web.reactive.function.client.WebClientResponseException;
+import com.linkedin.openhouse.relocated.reactor.core.publisher.Mono;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.compress.utils.Lists;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.Snapshot;
+import org.apache.iceberg.TableMetadata;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.exceptions.CommitStateUnknownException;
+import org.apache.iceberg.exceptions.NoSuchTableException;
+import org.apache.iceberg.io.FileIO;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class OpenHouseTableOperationsTest {
```

Comment:
why not reuse the source from other module if its unchanged?

```
sourceSets {
    test {
        java {
            srcDir project(':other-module').sourceSets.test.java.srcDirs
        }
        resources {
            srcDir project(':other-module').sourceSets.test.resources.srcDirs
        }
    }
}
```

Reply:
I'm using this trick now. Thanks.